boot/cypress/keys/cypress-test-ec-p256.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Any reason to make these keys specific to the cypress port, and not just use the keys at the root of the repo (which are intended for testing)?

Reply:
We supplied these key the same as are supplied within cysecuretools, so signed images will be consistent for al Cy-related Secure Boot FW.